scripts/print_cil_from_dn_file.py

```diff
@@ -52,11 +52,11 @@ def seek(self, offset: int) -> int:
 def read_dotnet_user_string(pe: dnfile.dnPE, token: StringToken) -> Union[str, InvalidToken]:
     """read user string from #US stream"""
     try:
-        user_string: Optional[dnfile.stream.UserString] = pe.net.user_strings.get_us(token.rid)
+        user_string: Optional[dnfile.stream.UserString] = pe.net.user_strings.get(token.rid)
```

Comment:
@malwarefrank is the type annotation still correct? It appears that `pe.net.user_strings.get` returns a `HeapItemString` now?

Reply:
type annotation should still be correct. `pe.net.user_strings` is a `UserStringHeap` and `.get()` returns a `UserString`, which is a subclass of `HeapItemString` and `HeapItemBinary`.  At least for now :)